routes/api/v1.php

```diff
@@ -22,19 +23,28 @@
 Route::get('/students', [StudentController::class, 'index'])->name('students.list');
 Route::get('/students/{id}', [StudentController::class, 'show'])->name('student.show');
 
+Route::post('/recruiters', [RecruiterController::class, 'store'])->name('recruiter.create');
+Route::get('/recruiters', [RecruiterController::class, 'index'])->name('recruiter.list');
```

Comment:
Could be a good idea to follow conventions 🤷‍♂️
```suggestion
Route::get('/recruiters', [RecruiterController::class, 'index'])->name('recruiter.index');
```